mautic/mautic
# Fix Timeline View Filters Broken issue

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Yes
| New feature? | No 
| Automated tests included? | No 
| Related user documentation PR URL | No
| Related developer documentation PR URL | No
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

The SalesForce UI Timeline Filters do not display correctly.  When viewing this particular timeline if you select a filter the screen contents distort and do not properly filter the data

![image](https://user-images.githubusercontent.com/48244990/111988561-abb20e80-8b36-11eb-9c0c-bf899e9786f2.png)


[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1.  It cannot be navigate from dashboard. It’s meant to be embedded via iframe to other systems  But you can go to the same route and use any contact ID at the end and should get to it.
2. use url {Instance URL}/s/plugin/Salesforce/timeline/view/{Contact ID}
Use contact ID that exists in your instance If you don’t have any then go to the Contacts menu item and create one.
3. Page becomes unresponsive and screen contents will be distort after selecting a filter.

#### Steps to test this PR:
1.  It cannot be navigate from dashboard. It’s meant to be embedded via iframe to other systems  But you can go to the same route and use any contact ID at the end and should get to it.
2. use url {Instance URL}/s/plugin/Salesforce/timeline/view/{Contact ID}
Use contact ID that exists in your instance If you don’t have any then go to the Contacts menu item and create one.
3. Filter should be works fine.
 

#### Other areas of Mautic that may be affected by the change:
1. 
2. 

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
